codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/customization/RemoveDefaults.java

```diff
@@ -32,7 +32,11 @@ public class RemoveDefaults implements GoIntegration {
         serviceToShapeIds("com.amazonaws.paymentcryptographydata#PaymentCryptographyDataPlane",
                 "com.amazonaws.paymentcryptographydata#IntegerRangeBetween4And12"),
         serviceToShapeIds("com.amazonaws.emrserverless#AwsToledoWebService",
-                "com.amazonaws.emrserverless#WorkerCounts"));
+                "com.amazonaws.emrserverless#WorkerCounts"),
+        serviceToShapeIds("com.amazonaws.imagebuilder#imagebuilder",
+                    // https://github.com/aws/aws-sdk-go-v2/issues/2734
```

Comment:
Thanks for actually putting the issue and ticket here, otherwise we'd be like 

"why do we need this customization?" 
"🤷"

Curious as to what happens when/if it gets fixed by the service. What happens with this customization?

Reply:
Nothing, it's a no-op for `ShapeBuilder.removeTrait`:

```
    public B removeTrait(ShapeId traitId) {
        if (this.traits.hasValue()) {
            ((Map)this.traits.get()).remove(traitId);
        }

        return this;
    }
```

Reply:
> "why do we need this customization?"

Currently asking myself this for every single other thing on this list, in fact!